beacon-chain/blockchain/core_test.go

```diff
@@ -564,6 +565,229 @@ func TestCanProcessAttestations(t *testing.T) {
 	}
 }
 
+func TestSaveProcessedBlock(t *testing.T) {
+	beaconChain, db := startInMemoryBeaconChain(t)
+	defer db.Close()
+
+	block := NewBlock(t, &pb.BeaconBlock{
+		SlotNumber:  64,
+		PowChainRef: []byte("a"),
+	})
+
+	hash, err := block.Hash()
+	if err != nil {
+		t.Error(err)
```

Comment:
Shouldn't it be `Fatal` here? If we can't hash a block, rest of the test will fail right?